src/site/_data/paths/learn-web-vitals.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "learn-web-vitals",
+    "cover": "/images/collections/web-vitals.svg",
+    "title": "Web Vitals",
+    "description": "Key signals of quality user experiences.",
```

Comment:
Change this to "Essential metrics for a healthy site" to keep it consistent with the current landing page.